packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -394,16 +403,24 @@ class _DatePickerDialogState extends State<DatePickerDialog> with RestorationMix
     Navigator.pop(context);
   }
 
+  void _handleOnDatePickerModeChange(){
+    if(widget.onDatePickerModeChange != null){
```

Comment:
space after the the `if`: `if (widget. ...)`

Reply:
Added space in if